### PR TITLE
morph/client: perform RPC switch and restore in one step

### PR DESCRIPTION
**Pull request by fyrchik**
Otherwise we could switch infinitely if subscription restore has failed.

Signed-off-by: Evgenii Stratonikov <evgeniy@nspcc.ru>

### CODE REVIEW

File: pkg/morph/client/client.go [hunk 1]
```diff
@@ -54,9 +54,9 @@ type Client struct {
 
 	cfg cfg
 
-	endpoints *endpoints
+	endpoints endpoints
 
-	// switching between rpc endpoint lock
+	// switchLock protects endpoints, inactive, and subscription-related fields.
```
Comment (carpawell) [inline pkg/morph/client/client.go:59]: is that change necessary? my idea was to clarify that mutex locks everything (no new RPCs until the switching process is finished not only protecting the fields). could you add that idea to the comment?

Comment (fyrchik) [reply pkg/morph/client/client.go:59]: Yes, because it protects shared data. Now it is obvious you can't use any of these fields without holding a mutex.
Added a comment about switching process.

**Review by carpawell — APPROVED** [review]

File: pkg/morph/client/fee.go [hunk 2]
```diff
@@ -33,19 +24,17 @@ func (x customFees) feeForMethod(method string) (fixedn.Fixed8, bool) {
...
 	defaultFee fixedn.Fixed8
 
-	customFees
-}
-
-// sets default fee for all operations.
-func (x *fees) setDefault(fee fixedn.Fixed8) {
-	x.defaultFee = fee
+	// customFees represents source of customized per-operation fees.
+	customFees map[string]fixedn.Fixed8
```
Comment (cthulhu-rider) [inline pkg/morph/client/fee.go:28]: Was the purpose of deprecating `customFees` type? If so, can we remove the type?
